examples/door-lock-app/door-lock-common/door-lock-app.matter

```diff
@@ -140,6 +140,70 @@ server cluster Basic = 40 {
   readonly attribute int16u clusterRevision = 65533;
 }
 
+server cluster ColorControl = 768 {
```

Comment:
Why do we have color control in a door lock?

Reply:
Yep disable this cluster in the zap file.

Reply:
Disabled